moviepy/editor.py

```diff
@@ -53,44 +77,25 @@
 
 try:
     from .video.io.sliders import sliders
+
+    __all__.append("sliders")
 except ImportError:
     pass
 
 # The next loop transforms many effects into VideoClip methods so that
 # they can be called with myclip.resize(width=500) instead of
 # myclip.fx( vfx.resize, width= 500)
-for method in [
-    "afx.audio_fadein",
-    "afx.audio_fadeout",
-    "afx.audio_normalize",
-    "afx.volumex",
-    "transfx.crossfadein",
-    "transfx.crossfadeout",
-    "vfx.crop",
-    "vfx.fadein",
-    "vfx.fadeout",
-    "vfx.invert_colors",
-    "vfx.loop",
-    "vfx.margin",
-    "vfx.mask_and",
-    "vfx.mask_or",
-    "vfx.resize",
-    "vfx.rotate",
-    "vfx.speedx",
-]:
-
-    exec("VideoClip.%s = %s" % (method.split(".")[1], method))
-
-
-for method in [
-    "afx.audio_fadein",
-    "afx.audio_fadeout",
-    "afx.audio_loop",
-    "afx.audio_normalize",
-    "afx.volumex",
-]:
-
-    exec("AudioClip.%s = %s" % (method.split(".")[1], method))
+audio_fxs = inspect.getmembers(afx, inspect.isfunction)
+video_fxs = itertools.chain(
+    inspect.getmembers(vfx, inspect.isfunction),
+    inspect.getmembers(transfx, inspect.isfunction),
+    audio_fxs,
+)
```

Comment:
Wouldn't it be simpler to just do 

    video_fxs = (
        audio_fxs
        + inspect.getmembers(vfx, inspect.isfunction)
        + inspect.getmembers(transfx, inspect.isfunction)
    )

`itertools.chain()` seems unnecessary.